app/controllers/reviews_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ReviewsController < ApplicationController
   def index
-    @reviews = Review.all
+    @food_truck = FoodTruck.find(params[:food_truck_id])
+    @reviews = @food_truck.reviews
   end
 end
```

Comment:
You don't have a route to reviews#index, nor do you have a view for the index page for reviews. You can delete this controller.